### PR TITLE
Update version info representation

### PR DESCRIPTION
**Pull request by divshacker**
# Changes made

Changed the last cell to represent the Version table in a better way. More  Details can be found in #1121 
# Justification

I changed the last cell to make it work.

### CODE REVIEW

**Review by frankharkins — CHANGES_REQUESTED** [review]
Looks like something funky going on with the image files. I'm not sure what's causing it but maybe try checking out these images from the master branch and committing again?

File: content/ch-algorithms/quantum-counting.ipynb [hunk 1]
```diff
@@ -138,7 +138,7 @@
     "import math\n",
     "\n",
     "# importing Qiskit\n",
-    "import qiskit\n",
+    "import qiskit.tools.jupyter\n",
```
[the remark this answers is not in the document]
Comment (divshacker) [reply content/ch-algorithms/quantum-counting.ipynb]: Done! Thanks

**Review by frankharkins — APPROVED** [review]
Looks good, thanks!

File: content/ch-states/representing-qubit-states.ipynb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{
 {
```
Comment (frankharkins) [inline content/ch-states/representing-qubit-states.ipynb:2]: This could be the culprit here

**Review by frankharkins — CHANGES_REQUESTED** [review]

File: content/ch-states/representing-qubit-states.ipynb (hunk 2 above)
Comment (divshacker) [reply content/ch-states/representing-qubit-states.ipynb:2]: Yes I was thinking same

**Review by frankharkins — APPROVED** [review]
Thank you!

**Pull request merged**